dev/tools/gen_defaults/lib/switch_template.dart

```diff
@@ -202,6 +202,19 @@ class _SwitchConfigM3 with _SwitchConfig {
 
   @override
   double get trackWidth => ${tokens['md.comp.switch.track.width']};
+
+  // The thumb size at the middle of the track. Hand coded default based on the animation specs.
+  // https://direct.googleplex.com/#/spec/283220003&285130003
```

Comment:
Is this a publicly available link? If not we probably shouldn't include it.

Reply:
+1

Reply:
I see. Removed it. Thanks!